Add check option to cli (#20)
kieran-ryan/pyprojectsort: README.md

pyprojectsort ../pyproject.toml
```

### Check formatting

To check whether formatting would be applied to your file you may use the **--check** option.

```console
pyprojectsort --check
```

> In case the file would be reformatted you get the output message ```'{file_path}' would be reformatted``` and the program terminates with exit code 1.
> 
> If the given file is in a standardised style you get the output message ```'{file_path}' would be left unchanged``` and the program terminates successfully.

Reviewer: Small suggested change to text:

```suggestion
When the file would be reformatted, the output message `'{file_path}' would be reformatted` displays and the program terminates with exit code 1.

If the given file matches the style of `pyprojectsort`, the output message `'{file_path}' would be left unchanged` displays and the program terminates successfully.
```
